File: Button/app/src/main/java/rob/myappcompany/button/MainActivity.java
package rob.myappcompany.button;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;

public class MainActivity extends AppCompatActivity {


    public void clickFunction(View view){

        Log.i("Info", "Button Pressed");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: KotlinFun/app/src/main/java/rob/myappcompany/kotlinfun/MainActivity.kt
 package rob.myappcompany.kotlinfun

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

 class MainActivity : AppCompatActivity() {

     var count = 0
     var textView = findViewById<TextView>(R.id.textView)

     fun reset(view : View){
         count = 0
         textView.text = count.toString()
     }

     fun plusOne(view : View){
         count++
         textView.text = count.toString()
     }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        textView.text = "0"
    }
}